run/pubsub/main.py

```diff
@@ -25,14 +27,16 @@
 @app.route('/', methods=['POST'])
 def index():
     envelope = request.get_json()
+    logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+
     if not envelope:
         msg = 'no Pub/Sub message received'
-        print(f'error: {msg}')
+        logging.error(f'error: {msg}')
```

Comment:
To make sure I understand, why does Python need the logging library instead of simply relying on print to stdout?

Reply:
I thought this was the standard we would want to set. It doesn't need it though.

Reply:
Do you prefer just flushing the stdout?

Reply:
The goal on this one is to be simple, idiomatic, and that it behaves in Stackdriver as expected. I don't have any requirements on the specifics of the first two.

Reply:
sg. Reverted to original sample and added a flush for logs.